sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-modules/files/commonconfig-6.6

```diff
@@ -861,9 +861,12 @@ CONFIG_TASKSTATS=y
 CONFIG_TASK_IO_ACCOUNTING=y
 CONFIG_TASK_XACCT=y
 CONFIG_TCG_ATMEL=m
+CONFIG_TCG_TIS=m
+CONFIG_TCG_TIS_I2C=m
 CONFIG_TCG_TIS_I2C_ATMEL=m
 CONFIG_TCG_TIS_I2C_INFINEON=m
 CONFIG_TCG_TIS_I2C_NUVOTON=m
+CONFIG_TCG_TIS_SPI=m
```

Comment:
the CI failed
```
 ERROR: sys-kernel/coreos-modules-6.6.21::coreos failed (configure phase):
2024-03-15T19:58:51.4682912Z  *   Requested options not enabled in build:
2024-03-15T19:58:51.4683239Z  *     CONFIG_TCG_TIS_SPI
```
Most likely, because CONFIG_SPI is required for this flag. Maybe we can add I2C and SPI later on, if needed. Adding these two options seem to enable a kernel tree that might bulk up the size.